src/lib.rs

```diff
@@ -6,7 +6,7 @@
 //! > provides interactions to and participates in the Web of Things.
 //!
 //! The crate relies on [serde](https://docs.rs/serde) to serialize and deserialize and provides
-//! an high level builder to ensure a valid TD is built and a set of accessors to make easier to
+//! an high level [builder](builder::ThingBuilder) to ensure a valid TD is built and a set of accessors to make easier to
 //! operate with complex descriptions.
```

Comment:
I would wrap the line (FYI: in (neo)vim, go on line 9 and then to `gqj` to "format this line and the following")
```suggestion
//! an high level [builder](builder::ThingBuilder) to ensure a valid TD is built and a set of
//! accessors to make easier to operate with complex descriptions.
```

Reply:
Folded in as well.